CONTRIBUTING.md

```diff
@@ -101,6 +101,64 @@ stories/core/**Alert.stories.tsx**
 
 - Contains common demo states and examples for the component.
 
+### Documentation
+
+We use [JSDoc](https://jsdoc.app/) to write documentation for our code.
+
+Every component should have a multiline description and at least one example.
+
+```jsx
+/**
+ * A small box to quickly grab user attention and communicate a brief message.
+ * @example
+ * <Alert>This is a basic alert.</Alert>
+ */
+export const Alert = (props: AlertProps) => {
+  ...
+```
+
+```jsx
+/**
+ * Footer element with all needed legal and info links.
+ * Be sure to place it manually at the bottom of your page.
+ * You can use position 'absolute' with relative body or set the height of the content and place footer at the end.
+ * @example <caption>Appending custom element after default elements</caption>
+ * <Footer customElements={[{title: 'Bentley', url: 'https://www.bentley.com/'}]} />
+ * @example <caption>Returning only custom elements</caption>
+ * <Footer customElements={() => newFooterElements)} />
+ * @example <caption>Filtering out a specific element</caption>
+ * <Footer customElements={(defaultElements) => defaultElements.filter(({ key }) => key !== 'privacy' )} />
+ * @example <caption>Changing a url</caption>
+ * <Footer customElements={(defaultElements) => defaultElements.map(element => ({ ...element, url: element.key === 'privacy' ? customPrivacyUrl : element.url }))} />
+ */
+export const Footer = (props: FooterProps) => {
+  ...
+```
+
+Every prop should have a multiline description with relevant informational tags.
```

Comment:
This line could even be combined with the above. i.e. `Each component and prop should have...`

Reply:
I changed how it's set up and don't think this makes sense anymore.